File: Source/Aladdin.ASN1.ISO/ASN1/ISO/PKIX/RevokedCertificates.cs
using System;
using System.Runtime.Serialization;

// RevokedCertificates ::= SEQUENCE OF RevokedCertificate

namespace Aladdin.ASN1.ISO.PKIX
{
	[Serializable]
	public class RevokedCertificates : Sequence<RevokedCertificate>
	{
		// конструктор при сериализации
        protected RevokedCertificates(SerializationInfo info, StreamingContext context) : base(info, context) {}

		// конструктор при раскодировании
		public RevokedCertificates(IEncodable encodable) : base(encodable) {} 

		// конструктор при закодировании
		public RevokedCertificates(params RevokedCertificate[] values) : base(values) {} 
	}
}

File: Source/Aladdin.ASN1.ISO.Java/src/aladdin/asn1/iso/pkix/RevokedCertificates.java
package aladdin.asn1.iso.pkix;
import aladdin.asn1.*; 
import java.io.*; 

// RevokedCertificates ::= SEQUENCE OF RevokedCertificate

public final class RevokedCertificates extends Sequence<RevokedCertificate>
{
    private static final long serialVersionUID = -5959341407700025141L;
    
	// конструктор при раскодировании
	public RevokedCertificates(IEncodable encodable) throws IOException 
	{
		super(RevokedCertificate.class, encodable); 
	} 
	// конструктор при закодировании
	public RevokedCertificates(RevokedCertificate... values) 
	{
		super(RevokedCertificate.class, values); 
	} 
}
